INSTALL.md

```diff
@@ -42,125 +54,151 @@ _**NOTE:** if you are updating Go from a older version, make sure you remove
 export VERSION=1.16.6 OS=linux ARCH=amd64  # change this as you need
 
 wget -O /tmp/go${VERSION}.${OS}-${ARCH}.tar.gz \
-  https://dl.google.com/go/go${VERSION}.${OS}-${ARCH}.tar.gz && \
+  https://dl.google.com/go/go${VERSION}.${OS}-${ARCH}.tar.gz
 sudo tar -C /usr/local -xzf /tmp/go${VERSION}.${OS}-${ARCH}.tar.gz
 ```
 
 Finally, set up your environment for Go:
 
 ```sh
-echo 'export GOPATH=${HOME}/go' >> ~/.bashrc && \
-  echo 'export PATH=/usr/local/go/bin:${PATH}:${GOPATH}/bin' >> ~/.bashrc && \
-  source ~/.bashrc
+echo 'export GOPATH=${HOME}/go' >> ~/.bashrc
+echo 'export PATH=/usr/local/go/bin:${PATH}:${GOPATH}/bin' >> ~/.bashrc
+source ~/.bashrc
 ```
 
 ## Install golangci-lint
 
-This is an optional (but highly recommended!) step. To ensure consistency and to
-catch certain kinds of issues early, we provide a configuration file for
-`golangci-lint`. Every pull request must pass the checks specified there, and
-these will be run automatically before attempting to merge the code. If you are
-modifying Singularity and contributing your changes to the repository, it's
-faster to run these checks locally before uploading your pull request.
+If you will be making changes to the source code, and submitting PRs, you should
+install `golangci-lint`, which is the linting tool used in the SingularityCE
+project to ensure code consistency.
+
+Every pull request must pass the `golangci-lint` checks, and these will be run
+automatically before attempting to merge the code. If you are modifying
+Singularity and contributing your changes to the repository, it's faster to run
+these checks locally before uploading your pull request.
 
 In order to download and install the latest version of `golangci-lint`, you can
 run:
 
 <!-- markdownlint-disable MD013 -->
+
 ```sh
 curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin
 ```
+
 <!-- markdownlint-enable MD013 -->
 
 ## Clone the repo
 
-Golang is a bit finicky about where things are placed. Here is the correct way
-to build Singularity from source:
+With the adoption of Go modules you no longer need to clone the SingularityCE
+repository to a specific location.
+
```

Comment:
Suggest removing this. Folks that have been around Go for a while already understand this, and for folks who haven't it isn't relevant...

```suggestion
```

Reply:
I put this in for now as it's relevant for people who have been around Singularity but don't know Go otherwise.

There have been instances where people trying to be helpful have suggested to others that our Admin guide is wrong, because it doesn't clone into GOPATH/src etc.

Agree we can remove it in future though.